src/libs/actions/Report.js

```diff
@@ -940,9 +940,16 @@ function addAction(reportID, text, file) {
  *  is last read (meaning that the entire report history has been read)
  */
 function updateLastReadActionID(reportID, sequenceNumber) {
+    // If we don't have a maxSequenceNumber for this report then we will assume that the report has just been created
+    // and use 0 for the maxSequenceNumber. Otherwise, we'll have an undefined sequenceNumber in cases where no
+    // sequenceNumber is explicitly passed.
+    const maxSequenceNumber = !_.isUndefined(reportMaxSequenceNumbers[reportID])
```

Comment:
Should this be an early return instead? Just thinking that if the `maxSequenceNumber` is `0`, then it doesn't really need to call `setLocalLastRead()` and make an API call.

Reply:
Oh hmm yeah I guess that would make sense wouldn't it 😄

Reply:
Probably a safer change as well since my comment above could end up being wrong at one point ...